### PR TITLE
Porting jenkins stuff from master to develop

### PR DESCRIPTION
**Pull request by mathom**

### CODE REVIEW

File: grunt/mocha_istanbul.js [hunk 1]
```diff
@@ -3,7 +3,8 @@ module.exports = {
         src: 'test/**/*.spec.js',
         options: {
             reporter: 'spec',
-            coverage: true
+            coverage: true,
+            reportFormats: ['cobertura', 'lcovonly']
```
Comment (tVeratti) [inline grunt/mocha_istanbul.js]: Cobertura is one of the reporters that comes with istanbul/nyc right?

**Comment by mathom** [comment]
[100% (0.0%) vs master 100%](https://jenkins.topbox.io/job/Topbox/job/rci-components/job/PR-605/1/)

File: grunt/mocha_istanbul.js (hunk 1 above)
Comment (troyblank) [reply grunt/mocha_istanbul.js]: Yea, it's a format it allows as a final result. All supported are `clover, cobertura, html, json-summary, json, lcov, lcovonly, none, teamcity, text-lcov, text-summary, text.`

Comment (mathom) [reply grunt/mocha_istanbul.js]: Cobertura is the format most Jenkins plugins understand so if you've got something to hook up, that's one to keep an eye out for.

**Pull request merged**